vm/src/pyobject.rs

```diff
@@ -252,6 +253,7 @@ impl AttributeProtocol for PyObjectRef {
             PyObjectKind::Module { name: _, ref dict } => dict.get_item(attr_name),
             PyObjectKind::Class { name: _, ref dict } => dict.get_item(attr_name),
             PyObjectKind::Instance { ref dict } => dict.get_item(attr_name),
+            PyObjectKind::List { elements: _ } => self.typ().get_attr(attr_name),
```

Comment:
Should raise ```AttributeError``` or possibly panic for the moment until this code returns a result.

Reply:
Agreed, reverted this line so it will panic.